tests/Pinch/ClientServerSpec.hs

```diff
@@ -34,9 +35,9 @@ echoServer = createServer $ \_ -> Just $ CallHandler $ \_ (r :: Value TStruct) -
   pure r
 
 data CalcRequest = CalcRequest
-  { inp1 :: Field 1 Int32
-  , inp2 :: Field 2 Int32
-  , op   :: Field 3 Op
+  { _inp1 :: Field 1 Int32
+  , _inp2 :: Field 2 Int32
+  , _op   :: Field 3 Op
```

Comment:
Hm, what's the point of this rename? Is this silencing some warnings?

Reply:
Yeah; there's some name shadowing, e.g. https://github.com/abhinav/pinch/blob/2578dc3114b443e6f1a19ca22f715edb38972001/tests/Pinch/ClientServerSpec.hs#L55

Personally, I would use record wild cards or something, but this was the easiest change